### ISSUE TITLE
Viewport3DX gets black when dragged back into draggable TabControl

### ISSUE THREAD

**Issue by AndreasReitberger** [helix-toolkit/helix-toolkit#1473]
Hi,
I'm using the `Dragablz` `TabablzControl` allowing to drag out tabs and dropped them back into the TabControl.#
https://github.com/ButchersBoy/Dragablz

Dragging out a tab with the Viewport3DX is working fine, however once I drop it back to the TabControl the view gets black and no content is shown anymore.

I guess it has something to do with the` EffectManager`?
I prepared a small testcase showing this issue.

https://github.com/AndreasReitberger/HelixToolkitDragablz

![image](https://user-images.githubusercontent.com/2952443/101240834-093e5e80-36f2-11eb-93a6-bc19b3d9d481.png)
![image](https://user-images.githubusercontent.com/2952443/101240840-14918a00-36f2-11eb-8643-f59f5919f1aa.png)
![image](https://user-images.githubusercontent.com/2952443/101240844-1b200180-36f2-11eb-8068-f26af402d352.png)

**Comment by sjc9999** [comment]
I'm having the same issue:
I'm using DevExpress WPF for .Net Core as my GUI toolkit.
I have implemented a UserControl control which contains a Viewport3DX view.  The user control is in a dockable panel.
Everything works fine when I undock panel but when I redock the graphics in the Viewport3DX view disappear and never come back.
I have verified that CompositionTargetEx.Rendering method is still being called.
I have tried calling the InvalidateArrange, InvalidateVisual, InvalidateSceneGraph, InvalidateRender, InvalidateMeasure, UpdateLayout methods on the Viewport3DX object when redocked.
I have looked at the panel content hierarchy between the UserControl and MainWindow when undocking and docking and it does change.
It is modified when undocking and restored back when redocking.
Any help in debugging this problem would be appreciated.
